MAUI/Scheduler/accessibility.md

```diff
@@ -0,0 +1,127 @@
+#  Accessibility Support in Maui Scheduler (SfScheduler)
+
+## Keyboard navigation
+
+The `SfScheduler` supports selection using keyboard interactions.
+
+#### Day, Week and WorkWeek views
+
+<table>
+<tr>
+<td>
```

Comment:
Add toc file for main title
https://github.com/syncfusion-content/maui-docs/blob/development/maui-toc.html

Reply:
Updated toc file for accessibility.